ipet/evaluation/IPETEvalTable.py

```diff
@@ -517,7 +542,7 @@ def tmpwrapper(*args):
             if not self.getName() in df_target:
                 df_target[self.getName()] = scalar
         
-        return df_long, df_target
+        return df_long, df_target, result
```

Comment:
If you change this, please update the doc of `getColumnData`

Reply:
You are right, I did commented on this now.